caps/templates/includes/council-powers.html

```diff
@@ -0,0 +1,154 @@
+{% if council.authority_type == 'CC' %}
+    <p class="mt-n1">{{ council.name }} has powers over:</p>
+{% elif council.country == council.NORTHERN_IRELAND %}
+    <p class="mt-n1">{{ council.name }} is a <strong>Northern Irish {{ council.get_authority_type_display }}</strong>, with powers over:</p>
+{% else %}
+    <p class="mt-n1">{{ council.name }} is a <strong>{{ council.get_authority_type_display }}</strong>, with powers over:</p>
+{% endif %}
+
+{% if 'staff' in council.powers %}
+    <details class="details details-accordion">
+        <summary class="mb-2">Council buildings and staff</summary>
+        <div class="council-powers">
+            <p>Climate actions might include:</p>
+            <ul>
+                <li>making council offices more energy efficient</li>
+                <li>incentivising ‘Active Travel’ or public transport use among employees</li>
+                <li>providing carbon literacy training for employees</li>
+            <ul>
+        </div>
+    </details>
+{% endif %}
```

Comment:
given this is always present we could skip the if, similarly with spending below?

Reply:
I like the consistency that every power has a conditional, even if it _does_ always evaluate to true in these two cases. Feels less surprising to look at the template and see everything nested in a check, rather than having two of them sit there without a condition.